oap-server/server-starter/src/main/java/org/apache/skywalking/oap/server/starter/OAPServerStartUp.java

```diff
@@ -18,9 +18,6 @@
 
 package org.apache.skywalking.oap.server.starter;
 
-/**
- * OAP starter
- */
 public class OAPServerStartUp {
```

Comment:
Considering ES7 starter removed, do you still need to keep this module? How about moving this into bootstrap directly?